common/cpp/react/renderer/components/rnscreens/RNSScreenShadowNode.h

```diff
@@ -4,13 +4,14 @@
 #include <react/renderer/components/rnscreens/EventEmitters.h>
 #include <react/renderer/components/rnscreens/Props.h>
 #include <react/renderer/components/view/ConcreteViewShadowNode.h>
+#include <jsi/jsi.h>
 
 namespace facebook {
 namespace react {
 
-extern const char RNSScreenComponentName[];
+JSI_EXPORT extern const char RNSScreenComponentName[];
 
-class RNSScreenShadowNode final : public ConcreteViewShadowNode<
+class JSI_EXPORT RNSScreenShadowNode final : public ConcreteViewShadowNode<
```

Comment:
I'd like to suggest defining
```cpp
#define RNSCREENS_EXPORT __attribute__((visibility("default")))
```
and using `RNSCREENS_EXPORT` instead.

Reply:
See: [2cfe611](https://github.com/software-mansion/react-native-screens/pull/1585/commits/2cfe611a7fc27467286f2733ea1f3a87d1958dcf)

I went with `JSI_EXPORT` macro as it handles not only `GNU` compiler attributes, but also  `MSVC`'s ones.